File: Modul6/Soal2/PRAK602-2410817120013-OktaviaLailaQodari.py
def main():
    jumlah_ruangan = int(input())
    angka = list(map(int, input().split()))
    
    hasil = []
    for i in range(jumlah_ruangan):
        jumlah = angka[i] * (i + 1)
        hasil.append(jumlah)
    
    print(" ".join(map(str, hasil)))

main()

File: Modul6/Soal2/PRAK602-2410817120013-OktaviaLailaQodari.c
#include <stdio.h>

int main() {
    int jumlah_ruangan;
    scanf("%d", &jumlah_ruangan);

    int angka[jumlah_ruangan];
    int hasil[jumlah_ruangan];

    for (int i = 0; i < jumlah_ruangan; i++) {
        scanf("%d", &angka[i]);
    }

    for (int i = 0; i < jumlah_ruangan; i++) {
        hasil[i] = angka[i] * (i + 1);
    }

    for (int i = 0; i < jumlah_ruangan; i++) {
        printf("%d ", hasil[i]);
    }
    printf("\n");

    return 0;
}